### ISSUE TITLE
Action button doesn't seems to have the right padding

### ISSUE THREAD

**Issue by kamendola** [adobe/react-spectrum#4433]
# 🐛 Bug Report 

`ActionButton` with `aria-label="Icon only"` appears to be `34px` by `32px` 

## 🤔 Expected Behavior

`ActionButton` with `aria-label="Icon only"` should be `32px` by `32px` according to the [designs specs ](https://spectrum.adobe.com/page/action-button/)

## 💻 Code Sample
https://react-spectrum.adobe.com/react-spectrum/ActionButton.html#accessibility

## 🧢 Your Company/Team

Adobe Experience Cloud, AEM, Universal Editor

**Comment by LFDanLu** [comment]
For context: `--spectrum-actionbutton-icon-padding-x` is returning a value that is 1 extra pixel too large in either direction. I'll have to dig to see if this was on purpose, but I think we should just override the var to return 6px instead of 7px OR update to follow the latest spectrum css for actionbutton

**Comment by snowystinger** [comment]
closing as not planned for v3
We'd still accept a contribution
